x-pack/plugins/maps/common/descriptor_types/descriptor_types.d.ts

```diff
@@ -121,20 +123,29 @@ export type SourceDescriptor =
   | ESGeoGridSourceDescriptor
   | EMSFileSourceDescriptor
   | ESPewPewSourceDescriptor
-  | TiledSingleLayerVectorSourceDescriptor;
+  | TiledSingleLayerVectorSourceDescriptor
+  | EMSTMSSourceDescriptor
+  | EMSFileSourceDescriptor;
+
+export type Query = {
```

Comment:
This is a duplicate of Kibana Query type, import { Query } from 'src/plugins/data/public'. Just use that instead.